tests/testthat/test-rr_journal_name.R

```diff
@@ -0,0 +1,33 @@
+context("rr_journal_name")
+
+test_that("rr_journal_name() works", {
+  skip_on_cran()
+
+  use_cassette("rr_journal_name", {
+    res = suppressWarnings({rr_journal_name("Journal of Geology")})
```

Comment:
I don't get any warning here

Reply:
it's true, me neither actually. I removed the call to `suppressWarnings()` in commit ff5f105